2009-02-06  Michael Tautschnig  <mt@debian.org>

	* lib/setup-storage/Commands.pm: Use the proper id for telling the user about
		preserved/resized partitions.
Index: trunk/lib/setup-storage/Commands.pm
===================================================================
--- trunk.orig/lib/setup-storage/Commands.pm
+++ trunk/lib/setup-storage/Commands.pm	
@@ -706,9 +706,9 @@
     $pre_all_resize .= ",exist_" . &FAI::make_device_name($disk, $p) unless
       $part->{size}->{resize};
     if ($part->{size}->{resize}) {
-      warn &FAI::make_device_name($disk, $p) . " will be resized\n";
+      warn &FAI::make_device_name($disk, $mapped_id) . " will be resized\n";
     } else {
-      warn &FAI::make_device_name($disk, $p) . " will be preserved\n";
+      warn &FAI::make_device_name($disk, $mapped_id) . " will be preserved\n";
       next;
     }
 
